test/views/templates/penalties/PenaltiesBannerSpec.scala

```diff
@@ -41,38 +41,158 @@ class PenaltiesBannerSpec extends ViewBaseSpec {
 
     "there are active penalties" when {
 
-      "there is one penalty point" should {
+      "there are only penalty points" when {
 
-        lazy val view = injectedView(model.copy(noOfPoints = 1, noOfCrystalisedPenalties = 0))
-        lazy implicit val document: Document = Jsoup.parse(view.body)
+        "there is one penalty point" should {
+
+          lazy val view = injectedView(model.copy(noOfPoints = 1, noOfCrystalisedPenalties = 0, noOfEstimatedPenalties = 0))
+          lazy implicit val document: Document = Jsoup.parse(view.body)
+
+          "have the correct heading" in {
+            elementText("h2") shouldBe "Late submission and late payment penalties"
+          }
+
+          "have the correct penalty information" in {
+            elementText(".govuk-notification-banner__content > div") shouldBe "Total penalty points: 1"
+          }
+
+          "have a link to the penalties service" which {
+
+            "has the correct text" in {
+              elementText("a") shouldBe "Find out why you have a penalty"
+            }
 
-        "have the correct heading" in {
-          elementText("h2") shouldBe "Late submission and late payment penalties"
+            "has the correct link destination" in {
+              element("a").attr("href") shouldBe mockConfig.penaltiesFrontendUrl
+            }
+          }
         }
 
-        "have the correct penalty information" in {
-          elementText(".govuk-notification-banner__content > div") shouldBe "Total penalty points: 1"
+        "there is more than one penalty point" should {
+
+          lazy val view = injectedView(model.copy(noOfPoints = 2, noOfCrystalisedPenalties = 0, noOfEstimatedPenalties = 0))
+          lazy implicit val document: Document = Jsoup.parse(view.body)
+
+          "have the correct penalty information" in {
+            elementText(".govuk-notification-banner__content > div") shouldBe "Total penalty points: 2"
+          }
+
+          "have the correct link text" in {
+            elementText("a") shouldBe "Find out why you have penalties"
+          }
+
+          "has the correct link destination" in {
+            element("a").attr("href") shouldBe mockConfig.penaltiesFrontendUrl
+          }
         }
 
-        "have a link to the penalties service" which {
+      }
+
+      "there are only crystalised penalties" when {
+
+        "there is one crystalised penalty point" should {
```

Comment:
Really minor thing here but this test and the one below it both mention "crystalised penalty point" which is fairly confusing. Can you remove the "point" from those descriptions